client/my-sites/plugins/main.jsx

```diff
@@ -775,9 +775,7 @@ export default React.createClass( {
 			);
 		}
 
-		if ( selectedSite &&
-				selectedSite.modulesFetched &&
-				! selectedSite.canManage() ) {
+		if ( selectedSite && ! selectedSite.canManage() ) {
```

Comment:
Just to note somewhere: `canManage` only exists on Jetpack sites, so we need to either check `selectedSite.jetpack` before calling it, or add the same method to a .com site in a way that makes sense. I'm not sure what the best solution is, but we pushed an update to check if the site is a Jetpack site before calling this to fix it in the meantime.